cpp/src/io/functions.cpp

```diff
@@ -194,7 +195,7 @@ void write_csv(csv_writer_options const& options, rmm::mr::device_memory_resourc
 namespace detail_orc = cudf::io::detail::orc;
 
 // Freeform API wraps the detail reader class API
```

Comment:
What is meant by "Freeform API"?

Reply:
Your guess is as good as mine :D 
Probably means that it consists of free functions. I'll remove this line.